hr_payroll_cancel/i18n/es_MX.po

```diff
@@ -0,0 +1,21 @@
+# Translation of OpenERP Server.
+# This file contains the translation of the following modules:
```

Comment:
.po files dont have a real translation
You could delete them and we can wait transifex